app/src/main/java/com/madonasyombua/sportsdb/ui/components/AppBar.kt

```diff
@@ -0,0 +1,21 @@
+package com.madonasyombua.sportsdb.ui.components
+
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+/**
+ * Created by Loveth Nwokike
+ * 3/22/2021
+ * */
+@Composable
+fun AppBar(title: String, actions: @Composable RowScope.() -> Unit = {}) {
```

Comment:
I am kind of wondering how one would  declare the full screen flags on` compose `